pegasus/sites.v3/hourofcode.com/public/how-to/districts.md.partial

```diff
@@ -13,26 +13,26 @@ nav: how_to_nav
 ## Get every school participating, every student learning!
 
 ### 1. Recruit local schools
-Share [this email](<%= resolve_url('/promote/resources#sample-emails') %>) and [teacher how to guide](<%= resolve_url('/how-to') %>), or include a [short blurb](<%= resolve_url('/promote/stats') %>) in newsletters/district communications.
+Share [this email]({{ promote/sample_emails}}) and [teacher how to guide]({{ urls/how_to_guide }}), or include a [short blurb]({{ promote/stats_url }}) in newsletters/district communications.
```

Comment:
nit: add a space between `promote/sample_emails` and the end curly brackets.

Reply:
Fixed, thanks